### PR TITLE
Protect against `undefind` in `from`

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: nodejs/query/sources.ts [hunk 1]
```diff
@@ -47,7 +47,9 @@ export function range(start: number, end?: number): AsyncIterable<number> {
 }
 
 export function from<TSource>(source: AsyncQuerySource<TSource>): AsyncIterable<TSource> {
-    return source instanceof Function
+    return source === undefined
```
Comment (justinvp) [inline nodejs/query/sources.ts]: Should we handle any falsy value in the same way (e.g. `null`)?

Comment (hausdorff) [reply nodejs/query/sources.ts]: I actually meant to protect specifically against spurious `undefined` (via accident-prone things like `!`) vs. `null` which I think tends to be intentional—you have to cast it to the type, I think, to pass it? We can do `null` too though

Comment (hausdorff) [reply nodejs/query/sources.ts]: What check did you have in mind for all falsey values?

Comment (justinvp) [reply nodejs/query/sources.ts]: > I actually meant to protect specifically against spurious `undefined` (via accident-prone things like `!`) vs. `null` which I think tends to be intentional—you have to cast it to the type, I think, to pass it? We can do `null` too though

From JavaScript you could pass anything...

> What check did you have in mind for all falsey values?

```typescript
return !source ? wrapAsyncIterable(() => []) : ...
```

I could also see throwing an error.

**Pull request merged**